srtcore/srt.h

```diff
@@ -531,6 +531,8 @@ SRT_API       int srt_bind         (SRTSOCKET u, const struct sockaddr* name, in
 SRT_API       int srt_bind_peerof  (SRTSOCKET u, UDPSOCKET udpsock);
 SRT_API       int srt_listen       (SRTSOCKET u, int backlog);
 SRT_API SRTSOCKET srt_accept       (SRTSOCKET u, struct sockaddr* addr, int* addrlen);
+typedef bool  srt_listen_callback_fn   (void* opaq, SRTSOCKET ns, int hsversion, const struct sockaddr* peeraddr, const char* streamid);
```

Comment:
Can't use 'bool' in C

Reply:
Well, that can be discussed (C99), but OTOH I see no problems with sticking to ANSI C here. Just probably I'd prefer then the "POSIX convention" (-1 for failure, 0 for success) rather than "boolean integer convention" (0 for failure, 1 for success). The first one would be consistent with the rest of the API. Tell me what you think.